src/main/resources/plugin.yml

```diff
@@ -1,5 +1,12 @@
 name: ServerTap
 main: io.servertap.PluginEntrypoint
 version: 1.0-SNAPSHOT
-api-version: 1.13
+api-version: 1.14
+loadbefore:
+  - Essentials
+  - Vault
+depend:
+  - Essentials
```

Comment:
@ATechAdventurer this hard dependency causes servertap to crash on startup if you don't have essentials. Can we move this to softdepend and also the `loadbefore` doesn't make sense to me - maybe I'm misunderstanding, but wouldn't we want server tap to always load _after_ the economy plugins?

Reply:
I didn't realize the essentials dependency had anything to do with the vault integration. It must have been a remnant of the old PR let me make that soft as well.

Reply:
@phybros I'm on mobile right now but I look through the code basin couldn't find any reference to essentials being used. So I'm really confused why it's even there in the first place.

Reply:
I think it was added in the original #7 that you took over, and is generally wrong. We don't need any sort of dep on Essentials.

I think your changeset basically just needs to add

```yaml
softdepend:
  - Vault
```

So the whole file ends up being

```yaml
name: ServerTap
main: io.servertap.PluginEntrypoint
version: 1.0-SNAPSHOT
api-version: 1.14
softdepend:
  - Vault

```